test/new-e2e/tests/npm/eks_1host_test.go

```diff
@@ -86,17 +91,17 @@ func eksHttpbinEnvProvisioner() e2e.PulumiEnvRunFunc[eksHttpbinEnv] {
 // TestEKSVMSuite will validate running the agent
 func TestEKSVMSuite(t *testing.T) {
 	t.Parallel()
+	provisionerOpts := []envkube.ProvisionerOption{}
```

Comment:
Shouldn't we remove it then as it's always empty? (and we can also pass inline as it uses varargs)

Reply:
Right, I no longer need that one. Deleted